.github/workflows/r.yml

```
@@ -233,14 +233,18 @@ jobs:
 
   windows-r:
     needs: [windows-cpp]
-    name: AMD64 Windows R RTools ${{ matrix.rtools }}
+    name: AMD64 Windows R ${{ matrix.config.rversion }} RTools ${{ matrix.config.rtools }}
     runs-on: windows-2019
     if: ${{ !contains(github.event.pull_request.title, 'WIP') }}
     timeout-minutes: 60
     strategy:
       fail-fast: false
       matrix:
-        rtools: [35, 40]
+        config:
+        - { rtools: 35, rversion: "3.6" }
+        - { rtools: 40, rversion: "4.1" }
+        # TODO: Once R 4.2 comes out we can switch to devel + 4.2
+        - { rtools: 40, rversion: "devel-ucrt" }
```

See https://github.com/r-lib/actions/blob/v2-branch/setup-r/src/installer.ts#L683-L686

Also we should test on the toolchain that CRAN uses. So we may continue to use rtools40 to build the C++ library but should use rtools42 for R >= 4.2.

```suggestion
        - { rtools: 42, rversion: "devel" }
```

---

Thanks!